### PR TITLE
Update open source community files

### PR DESCRIPTION
**Pull request by Jonasvdbo**
Updated the readme.md and added the following open source community files:

CONTRIBUTING.md
CODE_OF_CONDUCT.md
PROJECT_GOVERNANCE.md
RELEASE.md
SUPPORT.md

### CODE REVIEW

**Comment by Jonasvdbo** [comment]
@bramstoeller or @nitbharambe what you think of my PR?

**Review by bramstoeller — CHANGES_REQUESTED** [review]
Hi Jonas, thanks for adding these files. Could you please check all the references to power-grid-model-io. Some of them (still?) refer to power-grid-model. Also Aditya Hekker is mentioned in the council, but Tony Xiang took his place.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Ways of contributing
+
+Contribution does not necessarily mean committing code to the repository. 
+We recognize different levels of contributions as shown below in increasing order of dedication:
+
+1. Test and use the workshop. Give feedback on the user experience or suggest new features.
+2. Report bugs.
+3. Improve the workshop
+
```
Comment (nitbharambe) [inline CONTRIBUTING.md:20]: The ways of contribution will be slightly different here. Testing, using, new features, bugs, improvements will be present. Also,

- Add more conversions from other power system analysis libraries to power-grid-model

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# Getting Help
+
+There are a few ways to connect with the `power-grid-model` project:
+
+* Submit an issue
+* Send an email to the Technical Steering Committee: <dynamic.grid.calculation@alliander.com>.
+
+## How to ask for help, suggest a feature, or give recommendations.
+
```
Comment (nitbharambe) [inline SUPPORT.md:14]: This section would be different but similar to one in power-grid-model

https://github.com/alliander-opensource/power-grid-model/blob/main/docs/release_and_support/SUPPORT.md#how-to-ask-for-help-suggest-a-feature-or-give-recommendations

**Review by nitbharambe — CHANGES_REQUESTED** [review]
Thanks for adding the OS docs!

File: CONTRIBUTING.md (hunk 1 above)
Comment (Jonasvdbo) [reply CONTRIBUTING.md:20]: Good feedback. I will updat this.

**Comment by Jonasvdbo** [comment]
> Hi Jonas, thanks for adding these files. Could you please check all the references to power-grid-model-io. Some of them (still?) refer to power-grid-model. Also Aditya Hekker is mentioned in the council, but Tony Xiang took his place.

Good feedback. I will update this.

File: SUPPORT.md (hunk 2 above)
Comment (Jonasvdbo) [reply SUPPORT.md:14]: Good feedback. I will update this.

**Comment by Jonasvdbo** [comment]
The feedback has been processed. 

@nitbharambe Can you please re-review?

File: README.md [hunk 3]
```diff
@@ -3,8 +3,12 @@ SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dy
 
 SPDX-License-Identifier: MPL-2.0
 -->
+[![License: MIT](https://img.shields.io/badge/License-MPL2.0-informational.svg)](https://github.com/alliander-opensource/power-grid-model/blob/main/LICENSE)
```
Comment (nitbharambe) [inline README.md:6]: @Jonasvdbo I think Bram meant this by references.  Should point to power-grid-model-io

**Comment by Jonasvdbo** [comment]
Your remark regards the readme.md where I missed a reference to power grid model has been processed.

@nitbharambe Can you please re-review?

**Review by nitbharambe — APPROVED** [review]

**Comment by TonyXiang8787** [comment]
@Jonasvdbo please resolve the merge conflict

**Comment by Jonasvdbo** [comment]
Check. I see the readme.md has been changed in other PR from 2 hours ago. I solved the merge conflict.

**Comment by Jonasvdbo** [comment]
> @Jonasvdbo please resolve the merge conflict

@TonyXiang8787 Check. I see the readme.md has been changed in other PR https://github.com/alliander-opensource/power-grid-model-io/pull/44 from 2 hours ago. I solved the merge conflict.

**Comment by TonyXiang8787** [comment]
@Jonasvdbo  there are still build failures, can you have a look?

**Comment by Jonasvdbo** [comment]
> @Jonasvdbo there are still build failures, can you have a look?

@TonyXiang8787 Hmm, you are correct. I will have a look later today.

**Comment by Jonasvdbo** [comment]
@TonyXiang8787 The REUSE Comliance check is fixed. However, the Build, Test and Sonar check still fails. I don't fully understand why. Could have have a look what the problem could be?

**Comment by TonyXiang8787** [comment]
@Jonasvdbo the build of sonar cloud is a known issue from a PR in a forked repo. I will have a fix.

**Comment by Jonasvdbo** [comment]
> @Jonasvdbo the build of sonar cloud is a known issue from a PR in a forked repo. I will have a fix.

Ok, perfect!

**Review by TonyXiang8787 — APPROVED** [review]

**Review by bramstoeller — APPROVED** [review]

**Pull request merged**